src/ol/source/Cluster.js

```diff
@@ -29,7 +29,7 @@ import VectorSource from './Vector.js';
  * ```
  * See {@link module:ol/geom/Polygon~Polygon#getInteriorPoint} for a way to get a cluster
  * calculation point for polygons.
- * @property {VectorSource} source Source.
+ * @property {VectorSource} [source=null] Source.
```

Comment:
The type is already correct without your change: `[source]` means that it is optional.

Reply:
It should be changed to `[source]`, right?

Reply:
Right.